### PR TITLE
AFL variants new experiment request

### PR DESCRIPTION
**Pull request by jiradeto**
I'd like to request a new experiment to benchmark AFL variants which exploit a bunch of random approaches for fuzzing. There are 8 fuzzers that I add in this PR and their configurations are almost the same except for the environment variables that are set differently for each fuzzer.

### CODE REVIEW

**Comment by jiradeto** [comment]
Not sure why CI failed but I already called `make format` and the `make presubmit` was successful on my machine though.

File: fuzzers/afl_no_favs/fuzzer.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+
+def build():
+    """Build benchmark."""
+    afl_fuzzer.build()
+
+
+def fuzz(input_corpus, output_corpus, target_binary):
+    """Run fuzzer."""
+    os.environ['AFL_DISABLE_WRS'] ='1'
```
Comment (jonathanmetzman) [inline fuzzers/afl_no_favs/fuzzer.py:28]: Space after `=`. Same on next 4 lines.

**Review by jonathanmetzman — CHANGES_REQUESTED** [review]
Is it possible you didn't commit your changes?

File: fuzzers/afl_no_favs/fuzzer.py (hunk 1 above)
Comment (jiradeto) [reply fuzzers/afl_no_favs/fuzzer.py]: Adding space resolves the failing presubmit task. Thank you very much!

**Comment by jiradeto** [comment]
Hi @jonathanmetzman, I fixed the presubmit error but CI task `make RUNNING_ON_CI=yes -j test-run-afl-ffmpeg_ffmpeg_demuxer_fuzzer` seems to fail even though I did not touch any AFL setting. 

Am I missing something?

**Comment by jonathanmetzman** [comment]
> Hi @jonathanmetzman, I fixed the presubmit error but CI task `make RUNNING_ON_CI=yes -j test-run-afl-ffmpeg_ffmpeg_demuxer_fuzzer` seems to fail even though I did not touch any AFL setting.
> 
> Am I missing something?

Probably spurious.

**Comment by jonathanmetzman** [comment]
> Hi @jonathanmetzman, I fixed the presubmit error but CI task `make RUNNING_ON_CI=yes -j test-run-afl-ffmpeg_ffmpeg_demuxer_fuzzer` seems to fail even though I did not touch any AFL setting.
> 
> Am I missing something?

Probably spurious.

**Review by jonathanmetzman — APPROVED** [review]
LGTM

**Pull request merged**